common/entrypoint_mautic_web.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# wait untill the db is fully up before proceeding
+while [[ $(mysqladmin --host=db --user=$MYSQL_USER --password=$MYSQL_PASSWORD ping) != "mysqld is alive" ]]; do
+	sleep 1
+done
+
+# generate a local config file if it does't exist
+if [ ! -f /var/www/html/config/local.php ]; then
+
+	su -s /bin/bash www-data -c 'touch /var/www/html/config/local.php'
+
+	cat <<'EOF' > /var/www/html/config/local.php
```

Comment:
@ilkkao 

as the same image is uses for multiple purposes (`web`, `cron`  and `worker`), the file can't be copied during image generation, as the ` /var/www/html/config/` folder is mounted as a volume.

This could lead to a started container overwriting the file in the volume, thus losing your local config.

Does that make sense?

Reply:
Yes, I see. And users might modify this file (and add other files to the directory) so `/var/www/html/config/` needs to be a volume?

Reply:
indeed!